src/main/scala/io/qbeast/spark/internal/QbeastOptions.scala

```diff
@@ -135,14 +175,24 @@ object QbeastOptions {
     val txnAppId = getTxnAppId(options)
     val txnVersion = getTxnVersion(options)
     val userMetadata = getUserMetadata(options)
-    QbeastOptions(columnsToIndex, desiredCubeSize, stats, txnAppId, txnVersion, userMetadata)
+    val mergeSchema = getMergeSchema(options)
+    val overwriteSchema = getOverwriteSchema(options)
+    QbeastOptions(
```

Comment:
Perhaps not for this PR, but I'm wondering whether we should implement QbeastOptions following the builder pattern. It makes the creation of the object easier to maintain and visualize.

Reply:
Makes sense. Let me open an issue.